cli/cli.go

```diff
@@ -70,15 +68,15 @@ func newCli(
 			Value:  "127.0.0.1:42224",
 		},
 	)
-	nodeCreateOpts := model.NodeCreateOpts{
+	nodeCreateOpts := local.NodeCreateOpts{
```

Comment:
You'll notice that this is now more obviously coupled to a "local" node. Since this type wouldn't be used with a remote one, I think this makes a lot more sense.